.github/workflows/codespell.yaml

```diff
@@ -16,7 +16,7 @@ jobs:
       - name: codespell
         uses: codespell-project/actions-codespell@master
         with:
-          skip: .git
+          skip: .git,vendor
```

Comment:
also skip `go.mod` and `go.sum` ?

Reply:
sometimes we add comments in go.mod i can skip go.sum as its auto generated